bindings/java/pom.xml.in

```diff
@@ -36,6 +36,10 @@
   </scm>
 
   <properties>
+    <gpg.defaultKeyring>false</gpg.defaultKeyring>
+    <gpg.publicKeyring>${env.GPG_DIR}/pubring.gpg</gpg.publicKeyring>
+    <gpg.secretKeyring>${env.GPG_DIR}/secring.gpg</gpg.secretKeyring>
+    <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
```

Comment:
I've defined `GPG_PASSPHRASE` in pipelines. `.gpg` file names are the following: `sparkucx-public.gpg` and `sparkucx-secret.gpg`. You need to pull the first:
```yaml
  - task: DownloadSecureFile@1
    displayName: Download Secure file
    inputs:
      secureFile: sparkucx-secret.gpg
    name: privateKey
```

Azure doc on this step: [link](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-secure-file?view=azure-devops)

Reply:
This pipeline needs permission to access 2 resources before this run can continue to Build

Reply:
Granted permissions.